exercises/concept/captains-log/.docs/instructions.md

```diff
@@ -47,7 +47,12 @@ In the captain's log, stardates are usually rounded to a single decimal place.
 
 Implement the `format_stardate/1` function that will take a floating point number and return a string with the number rounded to a single decimal place.
 
+While there are valid approaches to this solution that might use an Elixir module, the implementation we are testing for expects an implementation using an Erlang function.
+
 ```elixir
 CaptainsLog.format_stardate(41458.15721310934)
 # => "41458.2"
+
+CaptainsLog.format_stardate(41411)
+# => ** (ArgumentError) argument error
```

Comment:
I appreciate where this coming from, but something we see often when we make those things too explicit is that a surprising number of students will implement those explicitly

```elixir
def format_stardate(n) when is_integer(n) do
  raise ArgumentError
end
```
which is not idiomatic and not easy to test for (although it could be an analyzer check).
Another point of view is that you wouldn't point out that `format_stardate("hi")` is an argument error.

So I think I would rather leave that out.